api/v1beta1/activemqartemis_types.go

```diff
@@ -224,7 +224,7 @@ type DeploymentPlanType struct {
 	// Whether or not to install the artemis metrics plugin
 	EnableMetricsPlugin *bool               `json:"enableMetricsPlugin,omitempty"`
 	Tolerations         []corev1.Toleration `json:"tolerations,omitempty"`
-	//custom labels provided in the cr
+	// Assign labels to a broker pod, the keys ActiveMQArtemis and application are not allowed
```

Comment:
if it is ok to use ` or ' in a comment, then 
 keys `ActiveMQArtemis` and `application` are not allowed

would make it more clear. But only a small point!

Reply:
quote added